src/models/stores.test.ts

```diff
@@ -0,0 +1,28 @@
+import { expect } from "chai";
+import { createStores } from "./stores";
+import { ProblemModel } from "./problem";
+import { UserModel } from "./user";
+
+describe("stores object", () => {
+
+  it("supports creating dummy stores for testing", () => {
+    const stores = createStores();
+    expect(stores).to.exist;          // tslint:disable-line
+    expect(stores.user).to.exist;     // tslint:disable-line
+    expect(stores.problem).to.exist;  // tslint:disable-line
+    expect(stores.ui).to.exist;       // tslint:disable-line
```

Comment:
Kind of curious, what was `tslint` complaining about here?

Reply:
You should be able to specify with something like `// tslint:disable-next-line:max-line-length
`

Reply:
I went ahead and added `tslint-no-unused-expression-chai`.